wicket-intro/CafeAddress/src/main/java/com/baeldung/wicket/examples/cafeaddress/CafeAddress.java

```diff
@@ -0,0 +1,72 @@
+package com.baeldung.wicket.examples.cafeaddress;
```

Comment:
This class is properly formatted, the same style you should use on the other classes and view templates. Usually, importing standard eclipse formatter to the IDE your using works out. 

Formatting we should apply on templates as well